### PR TITLE
Config file on bitrise script

### PR DESCRIPTION
**Pull request by felire**
add config file to run bitrise script, and I've added a way to run only the script without running the rest of the bootstrap

### CODE REVIEW

File: generators/app/bitriseUtils.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  const message = `The field ${key} in bitriseInfo.json file is required to run the script`;
+  if (
+    ((!value && typeof value === 'string') || value === null || value === undefined) &&
+    key !== 'projectPath' &&
+    key !== 'projectName'
+  ) {
+    return message;
+  }
+  return '';
+}
```
Comment (rodrigoWolox) [inline generators/app/bitriseUtils.js]: - Could value be any type different of string? If not you could just ask for `!value` and `key !==...`.
- You could make 
```
return condition ? `The field ${key} in bitriseInfo.json file is required to run the script` : '';
```

Comment (felire) [reply generators/app/bitriseUtils.js]: Yes, value could be boolean, so in that case I have to ask

File: generators/app/bitriseUtils.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    key !== 'projectPath' &&
+    key !== 'projectName'
+    ? `The field ${key} in bitriseInfo.json file is required to run the script`
+    : '';
+}
+
+function validateConfigObject(object) {
+  let lastMessage = '';
+  if (!object) {
+    return 'The bitriseInfo.json file is writed wrong';
```
Comment (WFolini) [inline generators/app/bitriseUtils.js]: `... is written wrong`

File: generators/app/bitriseUtils.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+function loadBitriseInfoFile() {
+  let configInfo = null;
+  try {
+    configInfo = JSON.parse(
+      fs.readFileSync('./wolmo-bootstrap-react-native/generators/app/bitriseInfo.json').toString()
+    );
+  } catch (e) {
+    console.log('The bitriseInfo.json file is writed wrong'.red.underline.bold);
```
Comment (WFolini) [inline generators/app/bitriseUtils.js]: Same

File: generators/app/tasks/bitriseInitialization.js [hunk 4]
```diff
@@ -10,98 +12,40 @@ module.exports = function bitriseInitialization() {
     context: this.options
   }).then(({ spinner }) => {
     spinner.stop();
-    return this.prompt([
```
Comment (WFolini) [inline generators/app/tasks/bitriseInitialization.js]: I'd keep all these prompt questions as a validation if there doesn't exist the file or if the key prop doesn't exist in the file config file. What u think? We can move them to `bitrise` or `bitriseUtils` files

Comment (felire) [reply generators/app/tasks/bitriseInitialization.js]: I don't know. I think it's not necessary. It's a lot of unnecessary code.

Comment (sfernandez11) [reply generators/app/tasks/bitriseInitialization.js]: I'm agree with Wally here.
If you move this code to another file we can create a command like `run.sh` to run the Bitrise script without clone the repo like we do here with the bootstrap: `bash <(curl -s https://raw.githubusercontent.com/Wolox/wolmo-bootstrap-react-native/master/run.sh)`.
If you delete these prompts you are forcing the user to clone the repo and fill the config file and not all users want this.
So, we can keep the config file if the user clone the repo and use the prompt questions if the user want to use the command above.
What do you think?

Comment (felire) [reply generators/app/tasks/bitriseInitialization.js]: Okey, I've finally did it. Please check

**Pull request merged**